client/src/store/index.ts

```diff
@@ -568,6 +571,21 @@ const {
       dispatch('reset');
       await djangoRest.logout();
     },
+    async loadGlobal({ commit }) {
+      const globalSettings = await djangoRest.globalSettings();
+      const pseudoProject = {
+        id: 'global',
+        name: null,
+        experiments: null,
+        settings: {
+          importPath: globalSettings.import_path,
+          exportPath: globalSettings.export_path,
+          permissions: null,
+        },
+        status: null,
+      };
+      commit('setCurrentProject', pseudoProject);
```

Comment:
Do we need this dummy project? What happens if we just have `currentProject = null`?

Reply:
The pseudo-project allows us to store the import and export paths, and it lets us avoid a major refactoring. It's possible to store them in a different place and refactor related files to use new checks. We check in a lot of places whether `currentProject` is null, for example, if `currentProject` is null we display "Select a Project" on the homepage. Should I make this change and refactor?

Reply:
Would the refactor be making import and export paths into computed properties to abstract this away? Or is there more to it?

Reply:
It just involves creating a new store variable and changing various `if` statements in the render. It's not too much so I made the change in [94b5f3a](https://github.com/OpenImaging/miqa/pull/294/commits/94b5f3a53811104e7720e22b1474cc59339bc2d3). Now we refer to global and project settings entirely separately.